src/styles/resetCSS.ts

```diff
@@ -1,6 +1,20 @@
 import { createGlobalStyle } from 'styled-components'
 
+import MikiRegular from '../fonts/Miki/KontrapunktMiki-Regular.otf'
+
 const GlobalStyle = createGlobalStyle`
```

Comment:
Hätte ich jetzt nicht hier erwartet. Als Global Style sollte richtig sein, so dass es in jeder React Component greift.

Reply:
Fixed